popSimManu.tex

```diff
@@ -726,11 +730,11 @@ \subsection*{Workflow for analysis of simulated data}
 using the HapmapII\_GRCh37 genetic map. For \textit{D.~melanogaster}
 we estimated $N(t)$ from an African sample simulated under the DroMel,
 African3Epoch\_1S16 model using the Comeron2012\_dm6 map.
-% RNG: No results from these simulations are included in the paper
-%Finally we ran simulations of \textit{A.~thaliana} genomes using the AraTha African2Epoch\_1H18 model under the Salome2012\_TAIR7 map.
-For each model, we estimated population size histories
-separately from each of three replicate simulations of whole genome datasets.
-In all cases we simulated 50 sampled chromosomes in the focal population.
+Finally we ran simulations of \emph{A. thaliana} genomes using the 
+AraTha African2Epoch\_1H18 model under the Salome2012\_TAIR7 map.
+For each model, three replicate whole genomes were
```

Comment:
might want to add the "demog_tuple" parameters for AraTha African2Epoch_1H18 (i.e. [500,1000])

Reply:
it's all specified in the catalog